packages/common/core-interfaces/Removing-IFluidRouter.md

```diff
@@ -90,7 +90,36 @@ const entryPoint = await container.getEntryPoint();
 
 ### Aliased DataStores
 
-(Not yet written)
+The `getRootDataStore` API has been deprecated on `IContainerRuntime` and `ContainerRuntime`. Migrate to usage of the new `IContainerRuntime.getAliasedDataStoreEntryPoint(...)` endpoint.
+The following is an example of that this change may look like:
+
+```ts
+// OLD
+const alias = "alias";
+const dataStore = await containerRuntime.createDataStore(TestDataObjectType);
+await dataStore.trySetAlias(alias);
+
+const retrievedDataStore = await containerRuntime.getRootDataStore(alias);
+```
+
+```ts
+// NEW
+const alias = "alias";
+const dataStore = await containerRuntime.createDataStore(TestDataObjectType);
+await dataStore.trySetAlias(alias);
+
+const entryPoint: IFluidHandle<ITestDataObject> | undefined =
+	await containerRuntime.getRootDataStore(alias);
+const retrievedDataStore = await entryPoint?.get();
+```
+
+### Sub-object routing
+
+Sub-object routing and requesting will continue to work as-is for the most part. The `IFluidDataStoreChannel` interface and `FluidDataStoreRuntime` class have a dedicated `request(...)` method, separate from the `IFluidRouter` concept.
```

Comment:
I don't think we should mention `IFluidDataStoreChannel` because its an internal interface and no one should care about it. It should not even have request. I believe the only reason it does is because data store context has a request method which calls it.

Reply:
We should remove the request method on data store context and hence from `IFluidDataStoreChannel`

Reply:
I agree. Ideally we want to cleanup all this `resolveHandle`/`request` stuff in regard to all the different data store related interfaces and classes.
[AB#6424](https://dev.azure.com/fluidframework/internal/_workitems/edit/6424)